modules/client-common/examples/01-extend-client.ts

```diff
@@ -0,0 +1,68 @@
+/* MARKDOWN
+---
+title: Extend client
+---
+
+## Extend client
+
+Use the ClientCore class to create an extended client. Now you can add custom functions and properties to the client.
+
+*/
+
+import { ClientCore, ContextCore } from "../src";
+
+// define a custom context params
+type CustomContextParams = {
+  customParam: string;
+};
+
+// define a custom context that extends the ContextCore class
+class CustomContext extends ContextCore {
```

Comment:
To avoid doubts, I would name any custom stuff like `My____`

```suggestion
class MyContext extends ContextCore {
```